core-web/libs/block-editor/src/lib/extensions/image-tabview-form/components/dot-image-card-list/dot-image-card-list.component.scss

```diff
@@ -0,0 +1,43 @@
+@use "variables" as *;
+
+:host {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    flex-direction: column;
+
+    .wrapper {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        flex-grow: 1;
+        height: 250px;
+        gap: 0.625 * $dot-editor-size;
+        padding: 0 (1.3125 * $dot-editor-size);
+        overflow: hidden;
+    }
+
+    .justify-start {
+        justify-content: flex-start;
+    }
+
+    &::ng-deep {
+        .p-scroller-content {
+            padding: 0 (1.3125 * $dot-editor-size);
+        }
+
+        .virtual-item,
+        .virtual-item-skeleton {
+            display: flex;
+            justify-content: space-between;
+            width: 100%;
+            min-height: 110px;
+        }
+
+        .virtual-item-skeleton {
+            justify-content: flex-start;
+            gap: $dot-editor-size;
+        }
+    }
```

Comment:
Any of this should go in global?

Reply:
I did a refactor. Also, don't think it's needed globally.